examples/step-68/doc/intro.dox

```diff
@@ -0,0 +1,120 @@
+<br>
+
+<i>
+
+Bruno Blais (Polytechnique Montréal),
+Toni El Geitani Nehme (Polytechnique Montreal),
+Rene Gassmöller (University of California Davis),
+and Peter Munch
```

Comment:
You wouldn't have to list me here. You did the work! But if you want to mention me:

```suggestion
and Peter Munch (Technical University of Munich and Helmholtz-Zentrum Geesthacht)
```